examples/text_classification_quantization_ort.ipynb

```diff
@@ -48,14 +48,14 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": 18,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "1.1.0\n"
+      "1.4.0.dev0\n"
```

Comment:
Should we also add a "Make sure your version of 🤗 Optimum is at least 1.4.0 "

Reply:
lets re run it after the launch again?